clevis/Sanity/kernel-keyring-support/PURPOSE

```diff
@@ -0,0 +1,3 @@
+PURPOSE of /CoreOS/clevis/Sanity/kernel-keyring-support
+Description: Test clevis to provide a parameter for the key description of a key in the kernel keyring
```

Comment:
Same comment as before regarding description

Reply:
This file won't go to the repo.